### DISCUSSION TITLE
Does 2.0.1 actually supoort String Schema for insert?

### DISCUSSION THREAD

**Discussion by AliscaCL** [milvus-io/milvus#15789] [Q&A and General discussion] [answered]
I have followed the tutoral to insert a DataType.STRING type schema into collections. 
But while I am actually inserting a string list into milvus, guess what? I had found that DataType.String inserting have not been implemented....
Is there any mistake in my code? Or my pymilvus version?
My Pymilvus version is 2.0.1, and milvus server is 2.0.1

**Comment by yhmo** [comment, marked as answer]
No, v2.0.1 doesn't support string type field.
Please wait for v2.1

**Reply by AliscaCL** [reply]
Okay~ That's fine. The v2.1 will come soon in this or next month? Really eager for that~

**Reply by yhmo** [reply]
We plan to release v2.1 RC1 on Apirl 14th, and official v2.1 on Apirl 26th.
https://wiki.lfaidata.foundation/display/MIL/Milvus+2.X+Roadmap+and+Time+schedule
